Package.swift

```diff
@@ -30,6 +30,9 @@ let package = Package(
             .product(name: "NIOWebSocket", package: "swift-nio"),
             .product(name: "NIOTransportServices", package: "swift-nio-transport-services"),
             .product(name: "Atomics", package: "swift-atomics")
+        ],
+        swiftSettings: [
+            .unsafeFlags(["-Xfrontend", "-strict-concurrency=complete"])
```

Comment:
I _think_ this should only affect people building this library but if I'm wrong we should remove it

Reply:
I couldn't even add `cSettings: [.unsafeFlags(["-Wno-redefined-macros"])]` to SQLiteNIO; I wouldn't risk it 😕